docs/.vuepress/config.js

```diff
@@ -560,6 +545,21 @@ module.exports = {
 						path: '/self-hosted/quickstart',
 						title: 'Quickstart',
 					},
+					{
+						type: 'page',
+						path: '/self-hosted/config-options',
+						title: 'Config Options',
+					},
+					{
+						type: 'page',
+						path: '/self-hosted/sso',
+						title: 'Single Sign-On (SSO)',
+					},
+					// {
+					// 	type: 'page',
+					// 	path: '/self-hosted/sso-examples',
+					// 	title: 'SSO Examples',
+					// },
```

Comment:
This page existed within the docs under config, but was not linked up via config.js
I moved it over to self-hosted, but kept this blocked out....
Lmk if we should have it visible or what.